mypy/constraints.py

```diff
@@ -949,7 +949,7 @@ def visit_instance(self, template: Instance) -> list[Constraint]:
             for item in actual.items:
                 if isinstance(item, UnpackType):
                     unpacked = get_proper_type(item.type)
-                    if isinstance(unpacked, TypeVarType):
+                    if isinstance(unpacked, (TypeVarType, TypeVarTupleType)):
```

Comment:
It can actually never be a regular `TypeVar`, it is just a typo that I forgot to test.
```suggestion
                    if isinstance(unpacked, TypeVarTupleType):
```